### PR TITLE
Update listOfProjects.js

### PR DESCRIPTION
**Pull request by Summer-source**

### CODE REVIEW

File: src/components/ProjectList/listOfProjects.js [hunk 1]
```diff
@@ -717,6 +717,14 @@ const projectList = [
     projectLink: 'https://github.com/zulip/zulip/contribute',
     description: 'Powerful Open Source Group Chat',
     tags: ['OpenSource', 'Chat', 'Javascript', 'Beginner', 'Python'],
-  }
+  },
+  {
+    name: 'Proton',
+    imageSrc: 'https://avatars1.githubusercontent.com/u/21955151?s=200&v=4',
+    githubLink:
```
Comment (kevinnls) [inline src/components/ProjectList/listOfProjects.js:724]:
```suggestion
   projectLink:

```

File: src/components/ProjectList/listOfProjects.js [hunk 2]
```diff
@@ -717,6 +717,14 @@ const projectList = [
...
+      'https:github.com/ValveSoftware/Proton.git',
```
Comment (kevinnls) [inline src/components/ProjectList/listOfProjects.js:725]:
```suggestion
      'https://github.com/ValveSoftware/Proton',
```

File: src/components/ProjectList/listOfProjects.js [hunk 3]
```diff
@@ -717,6 +717,14 @@ const projectList = [
...
+    description: 'A keyboard-driven, vim-like browser based on PyQt5',
```
Comment (kevinnls) [inline src/components/ProjectList/listOfProjects.js:726]: this does not seem to describe your linked project in any way :thinking: 

could you please check the link?

File: src/components/ProjectList/listOfProjects.js [hunk 4]
```diff
@@ -717,6 +717,14 @@ const projectList = [
...
+    tags: ['Python','Qt','windows','gamer','streamplay','qtWebEngine','app']
```
Comment (kevinnls) [inline src/components/ProjectList/listOfProjects.js:727]: i'm assuming from the link that you meant `SteamPlay` but again the other tags don't match the linked repo. there is only little Python and no Qt on Proton (as far as i can see)

File: src/components/ProjectList/listOfProjects.js [hunk 5]
```diff
@@ -717,6 +717,14 @@ const projectList = [
     projectLink: 'https://github.com/zulip/zulip/contribute',
     description: 'Powerful Open Source Group Chat',
     tags: ['OpenSource', 'Chat', 'Javascript', 'Beginner', 'Python'],
-  }
+  },
+  {
+    name: 'Proton',
+    imageSrc: 'https://avatars1.githubusercontent.com/u/21955151?s=200&v=4',
```
Comment (kevinnls) [inline src/components/ProjectList/listOfProjects.js:723]: that is qutebrowser's image. please change to Proton's logo

**Review by kevinnls — CHANGES_REQUESTED** [review]
i'm not sure about including this project. they don't have any issues labelled as `good first issue`. @Roshanjossey ?

as for the PR itself, i was initially quite confused but towards the end i think i understand you've copied the entry for a different project, but not changed all the values as required. so please make those changes

**Comment by sara-02** [comment]
@Summer-source Can you please look into the comments? Thanks

**Comment by kevinnls** [comment]
@Summer-source your PR has gained a lot of merge conflicts now; and the listing you added was mix-and-match. please provide the right details:
```js
{
    name: /*ITEM NAME*/,
    imageSrc: /*LINK TO ICON*/,
    projectLink: /*LINK TO PROJ GOOD FIRST ISSUES*/,
    description: /*DESCRIBE PROJ*/,
    tags: [/*tags*/, /*as*/, /*string*/, /*array*/]
  },
```
replace all the `/**/` comments with the right `'string values'`

i would recommend opening a fresh PR after syncing your fork with `source`

**Comment by Summer-source** [comment]
@kevinnls you can delete my pr . Thank You

**Comment by kevinnls** [comment]
have you opened a new one @Summer-source ?
please mention the new PR number

**Comment by Summer-source** [comment]
No i havent @kevinnls

**Pull request closed without merging**